src/clarity-angular/datagrid/_datagrid.clarity.scss

```diff
@@ -542,6 +543,11 @@
         }
     }
 
+    .datagrid-foot-description {
```

Comment:
The other components in datagrid use `.foot` instead of `.footer` and I wanted to be consistent with that. @youdz, can i change all of them to `.footer` instead?